### ISSUE TITLE
Some issues with `gk-deploy`

### ISSUE THREAD

**Issue by webwurst** [gluster/gluster-kubernetes#26]
Long arguments seem to have issues:
```
$ ./gk-deploy --deploy-gluster --verbose                                                                                               
Unknown option 'deploy-gluster'.

$ ./gk-deploy -g --verbose                                                                                                          1  
Unknown option 'erbose'.
```
Short versions work, but there seem to be some other issues:
```
./gk-deploy -g -v                                                                                                                    
Starting Kubernetes deployment.
serviceaccount "heketi-service-account" created
Found secret 'heketi-service-account-token-3juax' in namespace 'default' for heketi-service-account.
  File "<stdin>", line 10
    print node['node']['hostnames']['manage'][0]
             ^
SyntaxError: Missing parentheses in call to 'print'
Deploying GlusterFS pods on .
The Deployment "glusterfs-" is invalid: metadata.name: Invalid value: "glusterfs-": must match the regex [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)* (e.g. 'example.com')
Waiting for GlusterFS pods to start ... Checking status of pods matching 'glusterfs=pod':

Checking status of pods matching 'glusterfs=pod':
[...]
```

**Comment by webwurst** [comment]
`./gk-deply --abort` misses to delete deployment/pods with `--selector="glusterfs"`. See #27 for a possibly easy solution.

**Comment by jarrpa** [comment]
These should be fixed in #28.
